projects/go-lib/src/lib/components/go-accordion/go-accordion.component.ts

```diff
@@ -47,6 +47,7 @@ export class GoAccordionComponent implements OnInit, AfterContentInit {
       // NOTE: This feels a little destructive.
       // We lose track of the icon explicitly set by the child component.
       panel.icon = this.showIcons ? panel.icon : null;
+      panel.updateClasses();
```

Comment:
if we added the setter to the expanded class we would have to move the expanded property to be the last thing updated in this method